src/app/shared/twitter.service.ts

```diff
@@ -27,28 +26,26 @@ import { Observable } from 'rxjs';
 @Injectable()
 export class TwitterService {
   private TWITTER_SCRIPT_ID = 'twitter-wjs';
-  private TWITTER_WIDGET_URL = 'https://platform.twitter.com/widgets.js';
+  private TWITTER_WIDGET_URL = 'assets/mastodon-timeline.js';
```

Comment:
This is going to be gnarly, and I'm sorry, but I may have sent you off on the wrong track by experimenting and trying to get the current approach working with the `setTimeout` experiment, but I think the proper way to do this is as a new Angular component.

With Twitter, I believe we were required, both legally and technically, to load Twitter from their website, but we don't have that restriction here.

But if we make an Angular component, we shouldn't have to deal with all this asynchronous loading stuff and what not.